PCIe 6.0 Packet Generator and Exerciser/PCIe 6.0 Packet Generator and Exerciser/Endpoint/controller_class/configuration_controller.cpp

```diff
@@ -0,0 +1,171 @@
+#include "configuration_controller.h"
+
+/**
+ * @brief Construct a new Configuration Controller and initializing some needed pointers to be used inside it 
+ */
+ConfigurationController::ConfigurationController()
+{
+    configuration = ConfigurationSpace::constructConfigurationSpace();
+    capability =  PCIECapability::constructPCIECapability();
+
+    handler = make_shared<ConfigurationRequestHandler>(configuration, capability);
+    tlpConstructor = make_shared<TLPConstructor>();
+
```

Comment:
I understood that the TLP Constructor is something related to constructing the completion TLPs, so can you try to rename it to reduce ambiguity?

Reply:
Indeed, it's not the TLP itself (as you mentioned), but it's the class responsible for constructing the Completion TLPs, so I can't find another name for the class, but you can suggest a name.

Reply:
You Can name it as CompletionTLPConstructor maybe